packages/admin-ui/src/components/Filter/filter.tsx

```diff
@@ -0,0 +1,83 @@
+import type { ReactNode } from 'react'
+import React from 'react'
+
+import { Button } from '../Button'
+
+import { VisuallyHidden } from '../VisuallyHidden'
+import type {
+  FilterItem,
+  UseMultipleFilterReturn,
+} from './useMultipleFilterState'
+import { PopoverDisclosure } from 'reakit/Popover'
+import { PopoverFooter, Popover } from './popover'
+import { tag } from '@vtex/admin-ui-react'
+import { IconCaretUp } from '@vtex/phosphor-icons'
+import { MultipleSelectContent } from './multiple-select-content'
+
+export function Filter(props: FilterProps) {
+  const { state } = props
+  const { onClear, onChange, popover, selectedValues, label, labelProps } =
+    state
+
+  const selectedItemsLabel =
+    selectedValues.length &&
+    `${selectedValues[0]}${
+      selectedValues.length > 1 ? `, +${selectedValues.length - 1}` : ''
+    }`
+
+  return (
+    <>
+      <VisuallyHidden>
+        <div {...labelProps}>{label}</div>
+      </VisuallyHidden>
+      <Button
+        as={PopoverDisclosure}
+        state={popover}
+        csx={{
+          bg: '$secondary',
+          color: '$secondary',
+          ':hover': {
+            bg: '$action.neutral.secondaryHover',
+            color: '$secondary',
+          },
+          ':active': {
+            bg: '$disabled',
+            color: '$secondary',
+          },
+        }}
+        {...labelProps}
+      >
+        {label}
+        {!!selectedItemsLabel && (
```

Comment:
maybe it would be better to use directly `selectedValues.length > 1`  and `${selectedValues[0]}, +${selectedValues.length - 1}` to replace selectedItemsLabel.

Reply:
🤔